game/scripts/npc/abilities/ember_spirit_activate_fire_remnant.txt

```diff
@@ -51,7 +51,7 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "100 250 400 700 1300" //OAA
+        "damage"                                          "100 250 400 700 1000" //OAA
```

Comment:
I don't how plausible is to get to lvl 49 in competitive games. Ember was played in scrims before nerfs. Scaling 100/300/500/850/1200 was really op. At the major tournament he wasn't even picked. This is a pub nerf? If yes, hero is supposed to be really powerful at max level, thats the goal for every hero.

Reply:
Its a simillar issue that storm spirit had (Except emebr needs double the levels.). You don't have to play the game anymore, you just spam R around killing anyone in your path. 

I'll reduce the nerf to 1100 damag at maximum level. But I want embers other spells to deal damage. Smoothening the heroes powecurve.